tests/kernel_performance/install_qatestset.pm

```diff
@@ -0,0 +1,52 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+#
+# Summary: setup performance run environment
+# Maintainer: Joyce Na <jna@suse.de>
+
+package install_qatestset;
+use base "y2logsstep";
+use power_action_utils 'power_action';
+use strict;
+use warnings;
+use utils;
+use testapi;
+
+sub install_pkg {
+    my $qa_server_repo = get_var('QA_REPO');
+    sleep 10;
+    assert_script_run("hostname > /etc/hostname");
+    zypper_call("--non-interactive rr qa-ibs");
```

Comment:
No need for the additional `--non-interactive`, https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/814b3583ff6ef46d4cc591da5bb1cb16c6baa316/lib/utils.pm#L398 already includes that parameter. So simply write

```suggestion
    zypper_call'rr qa-ibs');
```

Reply:
updated